apps/parcel/src/app/ReactWidget/ReactWidget.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import singleSpaReact from 'single-spa-react';
+
+const ReactLogo = React.lazy(() => import('./ReactLogo'));
+
+class ReactWidget extends React.Component {
```

Comment:
The Angular dev in you is coming out here, in choosing to use a react class component instead of function component 😄  (this is a joke - using class components is completely fine)